examples/paywall-tester/src/main/res/layout/activity_paywall_footer_view.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context=".PaywallFooterViewActivity">
+
+    <LinearLayout
+            android:background="@android:color/holo_blue_light"
+            android:layout_weight="1"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:orientation="vertical"
+            android:gravity="center">
+        <TextView
+                android:text="Paywall Content"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"/>
+    </LinearLayout>
+
+    <com.revenuecat.purchases.ui.revenuecatui.fragment.PaywallFooterView
+            android:id="@+id/paywall_footer_view"
+            android:background="@android:color/holo_blue_light"
```

Comment:
Note how I'm setting the same background color as the view above here. This is so it looks like the view above continues behind the footer. But the positioning of the views will depend on the developer with this approach